eng/devices/windows.cake

```diff
@@ -168,61 +158,14 @@ Task("Test")
 	.IsDependentOn("SetupTestPaths")
 	.Does(() =>
 {
```

Comment:
I moved this code into powershell as all this code was doing was calling powershell - might as well make it a script and then I can actually read the msix, get the app id and certificate and all that. And it allowed the parser to somehow work because you can't have spaces or something in the `-Args` when starting the process.